mailu/values.yaml

```diff
@@ -61,9 +61,24 @@ mail:
 
 # certmanager settings
 certmanager:
+  enabled: false
+  # leave emty for default value
+  #apiVersion: cert-manager.io/v1alpha3
+  apiVersion:
```

Comment:
Please add the default here, that makes it more easy to see if it needs to be changed.